examples/scripts/ct_astra_unet_train_foam2.py

```diff
@@ -37,8 +37,8 @@
 Read data from cache or generate if not available.
 """
 N = 256  # phantom size
-train_nimg = 536  # number of training images
-test_nimg = 64  # number of testing images
+train_nimg = 498  # number of training images
```

Comment:
The notebook for this example should be re-run rather than just "updated".

Reply:
All notebooks were run to verify checkpointing.